client/app/routes/dashboards/dashboard.js

```diff
@@ -483,8 +483,11 @@ export default Ember.Route.extend({
                     {
                         chartType: 'donut',
                         widgetType: 'c3-chart',
-                        name: 'Publishers',
+                        name: 'Data providers',
```

Comment:
We should get clarification on renaming this widget since the chart shows where the results have been published, but the title "Data Providers" makes it sound like the chart is showing SHARE providers/sources.